### DISCUSSION TITLE
[Feature] Backup albums seperate of the main feed

### DISCUSSION THREAD

**Discussion by Quadrubo** [immich-app/immich#2492] [Feature Request]
### The feature

First of all sorry if this is kind of a duplicate of two separate features but I thought it deserved it's own post as most posts about hiding albums from the feed i've seen just deviated to archiving photos.

## Problem
I like to have my screenshots backed up but if I do so, they:
1. Mix with the camera photos so they become indistinguishable 
2. Clutter up the photos timeline

## Potential Solution
1. The albums that get chosen as being backed up by the mobile app get preserved and are stored separately from each other
2.  The user can decide which albums populate the main timeline

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#1678 & #5158
